source/jormungandr/jormungandr/realtime_schedule/synthese.py

```diff
@@ -69,6 +69,20 @@ def __repr__(self):
 class Synthese(RealtimeProxy):
     """
     class managing calls to timeo external service providing real-time next passages
+
+
+    curl example to check/test that external service is working:
+    curl -X GET '{server}?SERVICE=tdg&roid={stop_code}&rn={nb_desired}&date={datetime}'
+
+    {nb_desired} and {datetime} can be empty
+    {datetime} is on using format '%Y-%m-%d %H-%M' which is urlencoded (' ' > '%20')
+
+    On the response, Navitia matches route-point's {route_codes} (see details in _find_route_point_passages()).
+    {route_codes} and {stop_code} are provided using the same code key, named after
+    the 'destination_id_tag' if provided on connector's init, or the 'id' otherwise.
+
+    In practice it will look like:
+    curl -X GET 'http://bobito.fr/?SERVICE=tdg&roid=68435211116990230&rn=5&date=2018-06-11%2011:13'
```

Comment:
I think that this is more of an integration help and not a description of the class in itself. IMO, It shouldn't be here.